### DISCUSSION TITLE
Add privilege that user can get only role assigned to him.

### DISCUSSION THREAD

**Discussion by yoh1496** [personium/personium-core#650] [Ideas]
Related to personium/app-cc-home#263 .

`social-read` make user can read all roles about target cell.

**Comment by shimono** [comment]
This is important because social-read is too strong privilege to give to visitors in general.
First we need to design external spec.

**Comment by shimono** [comment]
Let us read https://datatracker.ietf.org/doc/html/rfc3744.html
and see if there are any suitable api to extend to achieve this goal.

**Reply by shimono** [reply]
This API looks good?
https://datatracker.ietf.org/doc/html/rfc3744.html#page-22
